lib/apr/notifications.ex

```diff
@@ -3,69 +3,70 @@ defmodule Apr.Notifications do
   require Logger
 
   def receive_event(event, topic, routing_key) do
-    event
-    |> slack_message(topic, routing_key)
-    |> post_message(topic, routing_key)
+    get_subscriptions(topic, routing_key)
```

Comment:
just to make sure, here we assume when someone subscribes to `commerce:fraud` since `fraud` is not routing key and its a _theme_, `routing_key` in this case will still be stored as `#` in the database for this subscriptions.

Reply:
We haven't wired up the theme parsing from the slack message yet. The event still gets triggered with the `topic` and `routing_key` as before and then we collect all the subscriptions relevant to those selections. The subscription will have the theme associated with it (which gets attached when parsing the slack command I assume).